### DISCUSSION TITLE
cpu.corespersocket

### DISCUSSION THREAD

**Discussion by mendyfasten** [apache/cloudstack#9877] [Q&A]
Hi, was wondering if anyone can point me in the right direction.
Have a Win 10 VM with 32 vCPU assigned. 
It only uses 16 cores, and Windows / Task Manager only shows 16 vCPU. It shows 4 Sockets and 16 Virtual processors.
I want to have 2 sockets and split the cpu's on them.
Is this possible?
Can't find the "cpu.corespersocket" setting although I've seen some chatter about it...

**Comment by weizhouapache** [comment]
@mendyfasten 
windows 10 supports max 4 cpu sockets, which could be the root cause.

you need to stop the vm at first, then go to the vm settings
![image](https://github.com/user-attachments/assets/3a2332d5-0630-44ac-a201-b3be52224acc)
try with 8 ?

**Comment by mendyfasten** [comment]
Thank you! This is what I was looking for. Had to set it to 16 per socket and it started using 32.

**Comment by mendyfasten** [comment]
https://github.com/apache/cloudstack/discussions/9877#discussioncomment-11108545
